client/src/app.tsx

```diff
@@ -20,7 +20,7 @@ import { SignIn, SignUp } from "./auth";
 import { Settings } from "./settings";
 
 import { DeepDives } from "./plus/deep-dives";
-import { Overview } from "./plus/deep-dives/overview";
+import { Overview as DeepDivesOverview } from "./plus/deep-dives/overview";
```

Comment:
This file is getting really busy as it is. The word `Overview` makes sense within `client/src/plus/deep-dives/overview/index.tsx` but nowhere outside.